scipy/stats/tests/test_multivariate.py

```diff
@@ -1430,6 +1430,15 @@ def test_invalid_dim(self):
         assert_raises(ValueError, ortho_group.rvs, 1)
         assert_raises(ValueError, ortho_group.rvs, 2.5)
 
+    def test_frozen_matrix(self):
+        dim = 7
+        frozen = ortho_group(dim)
```

Comment:
Should also test `ortho_group(dim, seed=1234)` and see that the first `rvs` it produces is the same as the others.